openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -240,9 +240,11 @@ function processIndicators(layerAttributes, projectDef) {
 
     // process each Social Vulnerability Index nodes
     // Get all the primary indicators
-    for (var i = 0; i < svThemes.length; i++) {
-        for (var e = 0 ; e < svThemes[i].children.length; e++ ) {
-            allPrimaryIndicators.push(svThemes[i].children[e].field);
+    if (svThemes) {
```

Comment:
This shouldn't be necessary, if https://github.com/gem/oq-platform/pull/442 is fixed